core/app-framework/app-native-shared/attr_container.c

```diff
@@ -8,10 +8,14 @@
 typedef union jvalue {
     bool z;
     int8_t b;
+    uint8_t q;
     uint16_t c;
     int16_t s;
+    uint16_t w;
```

Comment:
Had better exchange L12 and L13 to make the order consistent.

Reply:
The original field names are referred from Java, but it looks that it can meet the requirements now. How about changing them to:
```C
int8_t i8;
uint8_t u8;
int16_t i16;
uint16_t u16;
int32_t i32;
uint32_t u32;
int64_t i64;
uint64_t u64;
```

Reply:
Ok, I got it. I will rename it.